integration/solana/anchor/call_anchor.spec.ts

```diff
@@ -1,5 +1,5 @@
 import expect from 'expect';
-import { loadContract } from './setup';
+import { loadContract } from '../setup';
```

Comment:
Why are you moving this file to a sub directory?

Reply:
Anchor tests and Solana tests are semantically different and require a different preparation pipeline to run.

Reply:
So now we have some files in `integration/anchor` and the rest of them in `integration/solana/anchor`, for one "semantic" test.